app/src/main/java/com/example/composetemplate/presentation/common/LoginTextField.kt

```diff
@@ -91,9 +91,9 @@ fun LoginTextField(
 
         )
     // Automatically request focus and show the keyboard
-    LaunchedEffect(Unit) {
-        focusRequester?.requestFocus()
-        keyboardController?.show()
-    }
+//    LaunchedEffect(Unit) {
```

Comment:
Why is this in a comment? Is it relevant? If not, it's better to remove it.

Reply:
in the next PR It changed dont worry